lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -845,6 +845,11 @@ def install_puppetlabs_release_repo_on( hosts, repo = nil, opts = options )
             when /^(fedora|el|centos|sles|cisco)$/
               variant_url_value = (($1 == 'centos') ? 'el' : $1)
               variant_url_value = 'cisco-wrlinux' if variant == 'cisco'
+              if variant == 'cisco'
+                version = '5' if version == 'nexus'
+              else
+                version = '7' if version == 'ios_xr'
```

Comment:
I don't think you want the `else` here - you won't get to the `ios_xr` check with that in place.